### PR TITLE
Generate test coverage for PR checks.

### PR DESCRIPTION
**Pull request by omaciel**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [x] Refactor

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -10,17 +10,17 @@ export IQE_FILTER_EXPRESSION=""  # This is the value passed to pytest -k
 export IQE_CJI_TIMEOUT="30m"  # This is the time to wait for smoke test to complete or fail
 
 # Install bonfire repo/initialize
-CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
-curl -s $CICD_URL/bootstrap.sh > ${WORKSPACE}/cicd_bootstrap.sh && source ${WORKSPACE}/cicd_bootstrap.sh
+# CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
```
Comment (AaronH88) [inline pr_check.sh:13]: if these commented out bit are no longer required, please delete :smile:

**Comment by AaronH88** [comment]
Overall looks good, but its still tagged with do no merge, any reason for this?

**Comment by omaciel** [comment]
@AaronH88 I'm still investigating things and note that `pr_check.sh` is heavily commented out to speed up the testing steps. Sonarqube doesn't seem to be uploading or updating coverage results and I'm trying to get internal support.

**Pull request closed without merging**